smartbrain/k8s/worker-deployment.yaml

```diff
@@ -18,8 +18,14 @@ spec:
         ports:
         - containerPort: 2000
         env:
-          - name: REDIS_HOST
-            value: redis-service
+          - name: REDIS_HOST # ACRE Host Url
+            value: ""
+          - name: REDIS_PORT
+            value: "10000" # Default ACRE port
+          - name: REDIS_PWD # ACRE Key
+            value : ""
+          - name: REDIS_SSL
+            value : "1"
```

Comment:
i think for this repo, the value needs to be 0 by default as the user might not have an azure redis instance. i your scenario we should instruct them to change it to 1 since they will need it to use the azure cache for redis service